core/src/main/java/org/keycloak/representations/account/ClientRepresentation.java

```diff
@@ -22,4 +26,36 @@ public String getClientName() {
     public void setClientName(String clientName) {
         this.clientName = clientName;
     }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public boolean isInternal() {
+        return internal;
+    }
+
+    public void setInternal(boolean internal) {
+        this.internal = internal;
+    }
+
+    public boolean isInUse() {
+        return inUse;
+    }
+
+    public void setInUse(boolean inUse) {
+        this.inUse = inUse;
+    }
+
+    public String getUrl() {
+        return url;
+    }
+
+    public void setUrl(String url) {
+        this.url = url;
+    }
```

Comment:
@douglaspalmer we're missing created date and last access date. Marek commented on that thread about filtering on the mailing list. Let me quote him:

```
List<UserConsentModel> consents = session.users().getConsents(realmModel, userId);

Each UserConsentModel contains info about client, consent created date and last access date.
```

I noticed that you already have the code to retrieve Consents, maybe, it's just the matter of update.